testing/scenario_app/ios/Scenarios/Scenarios/TextPlatformView.m

```diff
@@ -92,6 +93,10 @@ - (instancetype)initWithFrame:(CGRect)frame
 }
 
 - (UIView*)view {
+  if (_viewCreated) {
```

Comment:
Nits: Let's add a comment here about why we needed this for feature reference :)

Reply:
@cyanglaz Done.